test/test_rotate.cpp

```diff
@@ -0,0 +1,119 @@
+#include <gtest/gtest.h>
```

Comment:
copyright also to this file

Reply:
Done in 8388e4148800e642fa919a0fe1ee945498747428